sxt/scalar25/operation/BUILD

```diff
@@ -45,10 +45,17 @@ sxt_cc_component(
         ":muladd",
         ":product_mapper",
         "//sxt/base/error:assert",
+        "//sxt/base/device:property",
         "//sxt/base/device:memory_utility",
+        "//sxt/base/iterator:index_range",
+        "//sxt/base/iterator:index_range_iterator",
+        "//sxt/base/iterator:index_range_utility",
         "//sxt/algorithm/reduction:reduction",
         "//sxt/memory/management:managed_array",
         "//sxt/memory/resource:async_device_resource",
+        "//sxt/execution/async:coroutine",
+        "//sxt/execution/device:device_viewable",
+        "//sxt/execution/device:for_each",
```

Comment:
It looks like buildifier didn't run. These are out of order.

Reply:
I don't think buildifier does sorting -- we might need to add that to our python script.